### PR TITLE
[KOGITO-3127] Set the `replaces` correctly in csv

### PR DESCRIPTION
**Pull request by radtriste**
https://issues.redhat.com/browse/KOGITO-3127

Csv fields are now corrected after the `make vet`
This is done as we may be coming from `1.0.0-snapshot`. latest released version is now taken from `deploy/olm-catalog/kogito-operator` directories.

Many thanks for submiting your Pull Request :heart:! 

Please make sure that your PR meets the following requirements:

- [x] You have read the [contributors' guide](https://github.com/kiegroup/kogito-cloud-operator/blob/master/README.md#contributing-to-the-kogito-operator)
- [x] Pull Request title is properly formatted: `[KOGITO-XYZ] Subject`
- [x] Pull Request contains link to the JIRA issue
- [x] Pull Request contains description of the issue
- [x] Pull Request does not include fixes for issues other than the main ticket
- [ ] Your feature/bug fix has a unit test that verifies it
- [x] You've tested the new feature/bug fix in an actual OpenShift cluster

### CODE REVIEW

**Review by sutaakar — APPROVED** [review]

**Review by ricardozanini — APPROVED** [review]
I know that we are in a hurry, but maybe a bats test validating this script wouldn't hurt. :D

**Comment by radtriste** [comment]
@ricardozanini gift to you :)

**Comment by radtriste** [comment]
For the bat test `check csv file is set correctly` I cannot do much faster as this depends on `operator-sdk` ...
and better to check everything is set correctly ;)

**Comment by radtriste** [comment]
I am on the correction of the test. Missing go and operator-sdk

**Comment by ricardozanini** [comment]
> For the bat test `check csv file is set correctly` I cannot do much faster as this depends on `operator-sdk` ...
> and better to check everything is set correctly ;)

:thinking: 

I'll write a service to validate CSV files on demand to make it easy for us.

File: .github/workflows/Kogito-Operator-PR-Check.yaml [hunk 1]
```diff
@@ -72,6 +72,27 @@ jobs:
     steps:
       - name: Checkout Code
         uses: actions/checkout@v2
+      - name: Setup Go ${{ env.GO_VERSION }}
+        uses: actions/setup-go@v2
+        with:
+          go-version: ${{ env.GO_VERSION }}
+        id: go
+      - name: Cache dependencies
```
Comment (ricardozanini) [inline .github/workflows/Kogito-Operator-PR-Check.yaml:80]: We don't need this since it's just a binary install

**Review by ricardozanini — APPROVED** [review]
I think we can remove the `cache dependencies` phase. This time will decrease considerably once we remove all those CRDs.

**Review by sutaakar — APPROVED** [review]

**Comment by radtriste** [comment]
> > For the bat test `check csv file is set correctly` I cannot do much faster as this depends on `operator-sdk` ...
> > and better to check everything is set correctly ;)
> 
> 
> 
> I'll write a service to validate CSV files on demand to make it easy for us.

The bats test is verifying the change we make in `bump-version.sh` is reflected and that the original version is not appearing anymore. Not checking the csv file itself ;)

File: .github/workflows/Kogito-Operator-PR-Check.yaml (hunk 1 above)
Comment (radtriste) [reply .github/workflows/Kogito-Operator-PR-Check.yaml:80]: removed

**Comment by ricardozanini** [comment]
> > > For the bat test `check csv file is set correctly` I cannot do much faster as this depends on `operator-sdk` ...
> > > and better to check everything is set correctly ;)
> > 
> > 
> > I'll write a service to validate CSV files on demand to make it easy for us.
> 
> The bats test is verifying the change we make in `bump-version.sh` is reflected and that the original version is not appearing anymore. Not checking the csv file itself ;)

It's unrelated topic that farted in my brain. Would be nice to validate the CSV as a service nevertheless.

**Pull request merged**